content/how-to/cloud-backup/cloud-backup-faq/index.md

```diff
@@ -253,10 +253,14 @@ account or **Administrative** access to Cloud Files for your sub-user account.
 
 {{< accordion title="What does Cloud Backup cost?" col="in" href="accordion17" >}}
 
+There are two costs. One is a Backup License Fee and other is cost of Storage
+which is for the Cloud File it uses to store customer data.
```

Comment:
I would recommend removing these two lines. There was likely a specific reason why mention of the license fee was redacted from this statement.

Reply:
Done. Many Thanks @LukeRepko .